### DISCUSSION TITLE
webinterface öffnet nicht

### DISCUSSION THREAD

**Discussion by Matten-Matten** [jomjol/AI-on-the-edge-device#2597] [General]
### Issue

Hallo ich habe ein problem beim einrichten meiner esp32 cam das webinterface öffnet nicht
und folgendes wird im Log ausgegeben:

```
I (31619) MAINCTRL: We are in Setup Mode -> Not starting Auto Flow!
I (41199) MAINCTRL: Autostart is not enabled -> Not starting Flow
W (179169) httpd_txrx: httpd_sock_err: error in send : 11
E (179169) SERVER HELP: File sending failed!
W (184169) httpd_txrx: httpd_sock_err: error in send : 11
W (184169) httpd_txrx: httpd_resp_send_err: 500 Internal Server Error - Failed to send file
W (189169) httpd_txrx: httpd_sock_err: error in send : 11
W (192969) httpd_uri: httpd_uri: uri handler execution failed
```

SD Karte neu formatiert kein - Erfolg
neue SD Karte frisch formatiert - kein Erfolg
ESP neu geflasht - kein Erfolg

was kann ich noch tun?

gruß Matten

**Comment by caco3** [comment]
hast Du mal einen anderen Webbrowser versucht?

**Reply by Matten-Matten** [reply]
ja auch das hab ich getestet

**Comment by Matten-Matten** [comment]
heute eine frisch gelieferte SD bekommen leider selber fehler. keine Ahnung was ich noch machen kann

**Comment by Matten-Matten** [comment]
muss mich berichtigen nach einem reboot dann plötzlich doch ein erreichen des Webinterface
